docker-compose.yml

```diff
@@ -267,7 +271,16 @@ services:
       - private
     volumes:
       - stackstorm-redis:/data
-
+  tester:
```

Comment:
Could we call it snpseq-tester or something? So that it is clear that it is something we have added.

Reply:
good suggestion, I'll fix